docs/resources/object_bucket.md

```diff
@@ -21,6 +21,83 @@ resource "scaleway_object_bucket" "some_bucket" {
 }
 ```
 
+### Using object lifecycle
+
+```hcl
+resource "scaleway_object_bucket" "main"{
+  name = "mybuckectid"
+  region = "fr-par"
+  acl = "private"
+  
+  # Rule case with storage classes GLACIER designed for low-cost data archiving
```

Comment:
Could you do a sentence in english that describe what this lifecycle does. For instance:

This rule will make all object that got a key that start with `path1/` be transferred from their default storage class (`STANDARD`, `ONEZONE_IA`) to GLACIER after 120 days counting from their creation and then 365 days after that they will be expired and deleted.